src/lib/SearchMap.js

```diff
@@ -179,10 +182,6 @@ export default class SearchMap {
      *  latitude values in a polar projection.
      */
```

Comment:
Is this callback executed when the user finishes drawing (the mouse is released), so that the comparison being done here is with the point where the user started their area selection?

Reply:
The callback actually happens with every mouse move, so the reason the direction is stored is so that it knows "are you dragging it across the dateline or did you go around the whole earth for real"; IE, whenever it runs, it checks the previous "direction" and coordinates to determine which way the user is dragging it.  When the user is done dragging and releases the mouse, it just uses whatever the latest information was.